data/json/recipes/recipe_food.json

```diff
@@ -3466,6 +3466,21 @@
     "tools": [ [ [ "surface_heat", 2, "LIST" ] ] ],
     "components": [ [ [ "powder_eggs", 2 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "fried_eggs",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
```

Comment:
I would lower the food handling to 1, so it is on the same level like "sloppy joes", "fried tofu" and "fried rice". Which is in my opinion harder to achieve than sunny side up.

Reply:
It should be a little more challenging than scrambled eggs imo. Ideally it should be food handling 1, but you'll fail a lot and just get scrambled eggs.